Packs/Akamai_WAF/Integrations/Akamai_WAF/Akamai_WAF.yml

```diff
@@ -348,13 +252,692 @@ script:
     - contextPath: Akamai.NetworkLists.ActivationStatus.ProductionStatus
       description: The network list environment activation status.
       type: String
-  dockerimage: demisto/akamai:1.0.0.13142
-  feed: false
-  isfetch: false
-  longRunning: false
-  longRunningPort: false
+  - arguments:
+    - description: The ID of the network list to update
+      name: network_list_id
+      required: true
+    - description: Comma separated list of elements. Use BLANK to empty a list.
+      isArray: true
+      name: elements
+      required: true
+    description: Updates list elements of a network list
+    name: akamai-update-network-list-elements
+  - arguments:
+    - name: checking_group_name
+      type: textArea
+    description: Check exist group within the context of your account.
+    name: akamai-check-group
+    outputs:
+    - contextPath: Akamai.check_group
+      description: Group ID
+    - contextPath: Akamai.check_group.Found
+      description: Found or not - Bool
+    - contextPath: Akamai.check_group.groupName
+      description: groupName
+    - contextPath: Akamai.check_group.parentGroupId
+      description: parentGroupId
+    - contextPath: Akamai.check_group.groupId
+      description: groupId
+    - contextPath: Akamai.check_group.checking_group_name
+      description: INPUT checking_group_name
+  - arguments:
+    - description: group_path - split with >
+      name: group_path
+      required: true
+    description: Create a new group under a parent gid
+    name: akamai-create-new-group
+  - arguments:
+    - defaultValue: US
+      description: Country
+      name: Country
+      required: true
+      type: textArea
+    - description: Company
+      name: Company
+      required: true
+      type: textArea
+    - description: Organizational Unit
+      name: OrganizationalUnit
+      required: true
+      type: textArea
+    - name: City
+      required: true
+    - description: Contract ID
+      name: contractId
+      required: true
+      type: textArea
+    - default: true
+      defaultValue: third-party
+      description: Certificate Type
+      name: certificateType
+      type: textArea
+    - description: Common Name
+      name: csr_cn
+      required: true
+    - description: Address of Admin Contact
+      name: adminContact_addressLineOne
+      required: true
+      type: textArea
+    - description: Firstname of Admin Contact
+      name: adminContact_firstName
```

Comment:
All input names should be separated with underscore - this should be named admin_contact_first_name